rotkehlchen/api/rest.py

```diff
@@ -6376,3 +6381,53 @@ def get_unmatched_asset_movements(self) -> Response:
         return api_response(_wrap_in_ok_result(
             result=list({event.group_identifier for event in asset_movements}),
         ))
+
+    def get_matches_for_asset_movement(
+            self,
+            asset_movement_group_identifier: str,
+            time_range: int,
+    ) -> Response:
+        events_db = DBHistoryEvents(database=self.rotkehlchen.data.db)
+        asset_movement = fee_event = None
+        with self.rotkehlchen.data.db.conn.read_ctx() as cursor:
+            for event in events_db.get_history_events_internal(
+                cursor=cursor,
+                filter_query=HistoryEventFilterQuery.make(group_identifiers=[asset_movement_group_identifier]),
+            ):
+                if isinstance(event, AssetMovement):
+                    if event.event_subtype == HistoryEventSubType.FEE:
+                        fee_event = event
+                    else:
+                        asset_movement = event
```

Comment:
what else can this be? wouldn't it be better to add the `entry_types` in the filter and ensure that you only have those?